### ISSUE TITLE
Incorrect xrefs between CMT4 and DHMN

### ISSUE THREAD

**Issue by cmungall** [DiseaseOntology/HumanDiseaseOntology#156]
I find some of these dubious (marked `???`). I am fairly sure the ones marked `***` are wrong.

```
[Term]
id: DOID:0050541
name: Charcot-Marie-Tooth disease type 4
namespace: disease_ontology
alt_id: DOID:0050582
def: "A Charcot-Marie-Tooth disease characterized by demyelinating or axonal abnormalities that has_material_basis_in autosomal recessive inheritance." [url:http://www.ninds.nih.gov/disorders/charcot_marie_tooth/detail_charcot_marie_tooth.htm]
comment: Xref MGI.nOMIM mapping confirmed by DO. [SN].
subset: DO_MGI_slim  ! DO_MGI_slim
subset: DO_rare_slim  ! DO_rare_slim
****synonym: "distal hereditary motor neuropathy" EXACT []
synonym: "hereditary motor and sensory neuropathy Russe type" EXACT []
synonym: "spinal CMT" EXACT []
???xref: OMIM:145900 ! Hypertrophic Neuropathy of Dejerine-Sottas
???xref: OMIM:158580 ! Neuronopathy, Distal Hereditary Motor, Type 7A
*****xref: OMIM:158590 ! Neuronopathy, Distal Hereditary Motor, Type 2A
xref: OMIM:182960 ! HMN1
xref: OMIM:214400 ! Charcot-Marie-Tooth Disease, Type 4A
???xref: OMIM:600794 ! Neuronopathy, Distal Hereditary Motor, Type 5A
xref: OMIM:601382 ! Charcot-Marie-Tooth Disease, Type 4B1
xref: OMIM:601455 ! Charcot-Marie-Tooth Disease, Type 4D
xref: OMIM:601596 ! Charcot-Marie-Tooth Disease, Type 4C
xref: OMIM:604563 ! Charcot-Marie-Tooth Disease, Type 4B2
???xref: OMIM:605253 ! Neuropathy, Congenital Hypomyelinating or Amyelinating, Autosomal Recessive
???xref: OMIM:605285 ! Neuropathy, Hereditary Motor and Sensory, Russe Type
>>>xref: OMIM:607641 ! Neuronopathy, Distal Hereditary Motor, Type 7B
xref: OMIM:607706 ! Charcot-Marie-Tooth Disease, Axonal, With Vocal Cord Paresis, Autosomal Recessive
???xref: OMIM:608634 ! Neuronopathy, Distal Hereditary Motor, Type 2B
xref: OMIM:609311 ! Charcot-Marie-Tooth Disease, Type 4H
xref: OMIM:611228 ! Charcot-Marie-Tooth Disease, Type 4J
???xref: OMIM:613376 ! Neuronopathy, Distal Hereditary Motor, Type 2C
???xref: OMIM:614751 ! Neuronopathy, Distal Hereditary Motor, Type 5B
xref: OMIM:614895 ! Charcot-Marie-Tooth Disease, Demyelinating, Type 4F
xref: OMIM:615284 ! Charcot-Marie-Tooth Disease, Type 4B3
?????xref: OMIM:615575 ! Neuronopathy, Distal Hereditary Motor, Type 2D
xref: Orphanet:64749
is_a: DOID:10595  ! Charcot-Marie-Tooth disease
```

Can you point to any evidence of CMT type 4 being associated with DHMN?

A very cursory search seems to reveal that CMT 2F is associated with DHMB 2A, see https://en.wikipedia.org/wiki/Distal_hereditary_motor_neuronopathies

If this is the case then the xref, if present at all, should be at the level of CMT2, not CMT4. Even here it seems dubious. Are we really meaning to have xref mean any of: equivalent, subclass, superclass,  phenotypically similar, caused by mutations in same gene, ....?

Here is the entry for CMT2; no link to DHMB 2A:

```
[Term]
id: DOID:0050539
name: Charcot-Marie-Tooth disease type 2
namespace: disease_ontology
def: "A Charcot-Marie-Tooth disease characterized by abnormalities in the axon of the peripheral nerve cell." [url:http://www.ncbi.nlm.nih.gov/pubmed/25098539, url:http://www.ninds.nih.gov/disorders/charcot_marie_tooth/detail_charcot_marie_tooth.htm]
comment: Xref MGI.nOMIM mapping confirmed by DO. [SN].
subset: DO_MGI_slim  ! DO_MGI_slim
subset: DO_rare_slim  ! DO_rare_slim
??? synonym: "hereditary motor and sensory neuropathy Guadalajara neuronal type" EXACT []
synonym: "hereditary motor and sensory neuropathy Okinawa type" EXACT []
???synonym: "hereditary motor and sensory neuropathy type 2" EXACT []
xref: ICD9:356.0
xref: OMIM:118210 ! Charcot-Marie-Tooth Disease, Axonal, Type 2A1
????xref: OMIM:118230 ! Charcot-Marie-Tooth Disease, Guadalajara Neuronal Type
xref: OMIM:600882 ! Charcot-Marie-Tooth Disease, Axonal, Type 2B
xref: OMIM:601472 ! Charcot-Marie-Tooth Disease, Axonal, Type 2D
xref: OMIM:604484 ! Neuropathy, Hereditary Motor and Sensory, Okinawa Type
xref: OMIM:605588 ! Charcot-Marie-Tooth Disease, Axonal, Type 2B1
xref: OMIM:605589 ! Charcot-Marie-Tooth Disease, Axonal, Type 2B2
xref: OMIM:606071 ! Hereditary Motor and Sensory Neuropathy, Type 2C
xref: OMIM:606595 ! Charcot-Marie-Tooth Disease, Axonal, Type 2F
xref: OMIM:607677 ! Charcot-Marie-Tooth Disease, Axonal, Type 2I
xref: OMIM:607684 ! Charcot-Marie-Tooth Disease, Axonal, Type 2E
xref: OMIM:607731 ! CMT2H
xref: OMIM:607736 ! Charcot-Marie-Tooth Disease, Axonal, Type 2J
xref: OMIM:607831 ! Charcot-Marie-Tooth Disease, Axonal, Type 2K
xref: OMIM:608591 ! CMT2G
xref: OMIM:608673 ! Charcot-Marie-Tooth Disease, Axonal, Type 2L
xref: OMIM:609260 ! Charcot-Marie-Tooth Disease, Axonal, Type 2A2
xref: OMIM:613287 ! Charcot-Marie-Tooth Disease, Axonal, Type 2N
xref: OMIM:614228 ! Charcot-Marie-Tooth Disease, Axonal, Type 2O
xref: OMIM:614436 ! Charcot-Marie-Tooth Disease, Axonal, Type 2P
xref: OMIM:615025 ! Charcot-Marie-Tooth Disease, Axonal, Type 2Q
xref: OMIM:615490 ! Charcot-Marie-Tooth Disease, Axonal, Type 2R
xref: OMIM:616155 ! Charcot-Marie-Tooth Disease, Axonal, Type 2S
xref: OMIM:616233 ! Charcot-Marie-Tooth Disease, Axonal, Type 2T
xref: OMIM:616280 ! Charcot-Marie-Tooth Disease, Axonal, Type 2U
xref: OMIM:616491 ! Charcot-Marie-Tooth Disease, Axonal, Type 2V
xref: Orphanet:64746
xref: SCTID:128202008
xref: SCTID:193158000
xref: SCTID:193159008
xref: SCTID:65017003
xref: UMLS:C0392553
is_a: DOID:10595  ! Charcot-Marie-Tooth disease
```

**Comment by pnrobinson** [comment]
May I suggest that we simply disregard the Disease-Ontology genetic diseases? DO should be importing from ORDO, OMIM or MONDO, and they simply are not doing a good job with this. In any case, I have looked into these terms. There following definitions of Charcot-Marie-Tooth (CMT) and CMT4 make the classification obvious and with one exception (!) we can just follow the primary term name of OMIM.



Charcot-Marie-Tooth (CMT) hereditary neuropathy refers to a group of disorders characterized by a chronic motor and sensory polyneuropathy. 

By convention, the designation CMT4 is applied to autosomal recessive forms of demyelinating Charcot-Marie-Tooth disease, which is a peripheral neuropathy characterized by distal motor and sensory impairment resulting in gait difficulties and associated with foot deformities. Motor nerve conduction velocities are decreased, and sural nerve biopsies show loss of myelinated fibers. The age at onset and severity is variable (summary by Patzko and Shy, 2012).

Genetic Heterogeneity of Charcot-Marie-Tooth Disease Type 4

Several different subtypes of autosomal recessive demyelinating CMT (CMT4) have been identified, each with particular ethnic, pathologic, or clinical characteristics: CMT4A; CMT4B, which includes CMT4B1 (601382), caused by mutation in the MTMR2 gene (603557), CMT4B2 (604563), caused by mutation in the SBF2 gene (607697), and CMT4B3 (615284), caused by mutation in the SBF1 gene (603560); CMT4C (601596), caused by mutation in the SH3TC2 gene (608206); CMT4D (601455), caused by mutation in the NDRG1 gene (605262); CMT4E (605253), caused by mutation in the EGR2 (129010) or MPZ (159440) genes; CMT4F (614895), caused by mutation in the PRX gene (605725); CMT4G, or Russe-type hereditary motor and sensory neuropathy, (605285), which maps to chromosome 10q23; CMT4H (609311), caused by mutation in the FGD4 gene (611104); CMT4J (611228), caused by mutation in the FIG4 gene (609390); and CMT4K (616684), caused by mutation in the SURF1 gene (185620).


 Xrefs 	

OMIM:145900
Dejerine-Sottas disease 	=> No

OMIM:158580
 	Neuronopathy, distal hereditary motor, type VIIA => NO

OMIM:158590
Neuropathy, distal hereditary motor, type IIA => NO (Charcot-Marie-Tooth disease type 2L (CMT2L; 608673) is an allelic disorder with an overlapping phenotype.)
=> Note that 158590 is NOT Charcot Marie Tooth because it is limited to Motor deficits (sensory not affected).

OMIM:182960
 Neuronopathy, distal hereditary motor, type I NO

OMIM:214400
Charcot-Marie-Tooth disease, type 4A => YES

OMIM:600794
Neuropathy, distal hereditary motor, type VA => NO

OMIM:601382
 	Charcot-Marie-Tooth disease, type 4B1 => YES

OMIM:601455
 	Charcot-Marie-Tooth disease, type 4D => YES

OMIM:601596
Charcot-Marie-Tooth disease, type 4C => YES

OMIM:604563
Charcot-Marie-Tooth disease, type 4B2 => YES

OMIM:605253
Neuropathy, congenital hypomyelinating  => NO

OMIM:605285
Neuropathy, hereditary motor and sensory, Russe type => YES (The disease is also known as Charcot-Marie-Tooth disease type 4G (CMT4G) ).

OMIM:607641
Neuropathy, distal hereditary motor, type VIIB => NO

OMIM:607706
Charcot-Marie-Tooth disease, axonal, with vocal cord paresis 	 => This is not a type 4 Charcot Marie tooth, so => NO
OMIM:608634
Neuropathy, distal hereditary motor, type IIB 	=> Not type 4 CMT

OMIM:609311
Charcot-Marie-Tooth disease, type 4H => YES

OMIM:611228
Charcot-Marie-Tooth disease, type 4J 	 => YES

OMIM:613376
Neuronopathy, distal hereditary motor, type IIC => NO

OMIM:614751
?Neuronopathy, distal hereditary motor, type VB => NO

OMIM:614895
Charcot-Marie-Tooth disease, type 4F => YES

OMIM:615284
Charcot-Marie-Tooth disease, type 4B3 	 => YES

OMIM:615575
Neuronopathy, distal hereditary motor, type IID => NO

ORDO:64749
Charcot-Marie-Tooth disease type 4 => YES

**Comment by elviram** [comment]
We will take a closer look at these.

**Comment by lschriml** [comment]
Hello Chris and Peter, 
   As I'm sure you have noticed we have been working with MGI and RGD to split out the genetic subtypes, and aligning these with OMIM. Previous to this area of work, OMIM terms were grouped in DO, as genetic subtypes were in the process of being defined. The examples you provided are from terms that have since been split. This is a multi-month process as we work to carefully to define the genetic subtypes (reviewing pertinent literature) and to assess the level of evidence for each disease. We did look to see if we could just adopt OMIM or Orphanet's categories, but after much evaluation, we found that we needed to review each disease term and manually define them. We are not importing MONDO, as it is built on DO and would not aid in this process. 
Pertinent to these two examples: 
Charcot-Marie-Tooth disease type 4:
is now subdivided into 12 subtypes. 
The OMIM IDs have been removed from type 4. 
synonyms have been reviewed and edited

Cheers,
Lynn

**Comment by pnrobinson** [comment]
?Hi Lynn,

we'd be happy to have more contacts & synergy on this topic. We are not producing rare disease hierarchies ourselves in MONDO, but rather importing and integrating various sources including DO, as you know. However, Orphanet is doing essentially what you are describing as ORDO -- I wonder if there could be some mutual benefit for your groups. Let me know if you would like an introduction.

Also, the upper level categories for genetic disease in DO need work -- the use of syndrome etc is inconsistent. I would be glad to advise.

-Peter


Peter Robinson

Professor of Computational Biology

The Jackson Laboratory for Genomic Medicine

10 Discovery Drive

Farmington, CT 06032

860.837.2095 t | 860.990.3130 m

peter.robinson@jax.org<mailto:peter.robinson@jax.org>

www.jax.org



The Jackson Laboratory: Leading the search for tomorrow's cures


________________________________

**Comment by cmungall** [comment]
> We are not importing MONDO, as it is built on DO and would not aid in this process

Obviously importing MonDO into DO makes no sense. However, you can use the kboom algorithm that constructs MonDO to resolve multiple alternate classifications and detect problem areas. This is where these requests from me to the DO tracker have been coming from. I didn't notice them myself by poring through the DO looking at xrefs, they fall out of the reporting. I spoke to @elviram about this at the BD2K meeting.
